Caculator_G/Caculator_G/ViewController.swift

```diff
@@ -11,21 +11,85 @@ import UIKit
 class ViewController: UIViewController {
```

Comment:
```suggestion
final class CaculatorViewController: UIViewController {
```

더 이상 상속되지 않는 class라면 `final`을 붙여서 표시하면 알기 쉽답니다 :)
가능하다면 ViewController이름도 명확하게 지어주면 좋을 것 같아요 ~